code/modules/hydroponics/gene_modder.dm

```diff
@@ -244,6 +266,40 @@
 
 	data["modal"] = ui_modal_data(src)
 
+	var/list/stats = list()
+	var/list/traits = list()
+	var/list/reagents = list()
+	var/empty_disks = 0
+
+	data["stat_disks"] = list()
+	data["trait_disks"] = list()
+	data["reagent_disks"] = list()
+
+	for(var/i in 1 to length(contents))
+		if(contents[i].type == /obj/item/disk/plantgene)
```

Comment:
also make this an early continue statement

Reply:
what do you mean by this?